src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public static void doWork() {
+        System.out.println("Bulldozer started its work");
+
+    }
+
+    public static void stopWork() {
+        System.out.println("Bulldozer stopped working ");;
```

Comment:
The message 'Bulldozer stopped working' could be more informative. For example, 'Bulldozer has finished moving earth and is now off'.